v20.1/views.md

```diff
@@ -365,6 +339,61 @@ To remove a view, use the [`DROP VIEW`](drop-view.html) statement:
 DROP VIEW
 ~~~
 
+## Temporary views
+
+<span class="version-tag">New in v20.1:</span> CockroachDB supports session-scoped temporary views. Unlike persistent views, temporary views can only be accessed from the session in which they were created, and they are dropped at the end of the session. You can create temporary views on both persistent tables and [temporary tables](temporary-tables.html).
+
+{{site.data.alerts.callout_danger}}
+**This is an experimental feature**. The interface and output are subject to change. For details, see the tracking issue [cockroachdb/cockroach#46260](https://github.com/cockroachdb/cockroach/issues/46260).
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_info}}
+Temporary tables must be enabled in order to use temporary views. By default, temporary tables are disabled in CockroachDB. To enable temporary tables, set the `experimental_enable_temp_tables` [session variable](set-vars.html) to `on`.
+{{site.data.alerts.end}}
+
+### Details
+
+- Temporary views are automatically dropped at the end of the session.
+- A temporary view can only be accessed from the session in which it was created.
+- Temporary views persist across transactions in the same session.
+- Temporary views cannot be converted to persistent views.
+- Temporary views can be created on both persistent tables and [temporary tables](temporary-tables.html).
+- When you create a view on a temporary table, the view automatically becomes temporary.
+
+{{site.data.alerts.callout_info}}
+Like [temporary tables](temporary-tables.html), temporary views are not in the `public` schema. Instead, when you create the first temporary table, view, or sequence for a session, CockroachDB generates a single temporary schema (`pg_temp_<id>`) for all of the temporary objects in the current session for a database.
+{{site.data.alerts.end}}
+
+### Usage
+
+To create a temporary view, add [`TEMP`/`TEMPORARY`](sql-grammar.html#opt_temp) clause to the beginning of a [`CREATE VIEW`](create-view.html) statement.
```

Comment:
I think this mention of the grammar clause is a bit confusing. It's arguably not needed given the example below.